File: Selenium/Activities/Activity12.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity12 {

	public static void main(String[] args) {

		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/dynamic-controls");

		System.out.println("The Title of the page is  : " + driver.getTitle());
		WebElement txtField = driver.findElement(By.id("input-text"));

		System.out.println("Text Box is Enabled : " + txtField.isEnabled());

		WebElement enableInputButton = driver.findElement(By.cssSelector("#toggleInput"));

		enableInputButton.click();

		System.out.println("Text Box is Enabled : " + txtField.isEnabled());
		driver.quit();

	}

}

File: Selenium/Activities/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC


with webdriver.Firefox() as ff:

    ff.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("title is : ", ff.title)
    wait = WebDriverWait(ff, 10)
    inputBox = ff.find_element(By.ID, "input-text")
    inputButton = ff.find_element(By.CSS_SELECTOR, "#toggleInput")

    print("Input text box is enabled : ", inputBox.is_enabled())

    inputButton.click()
    print("Input text box is enabled : ", inputBox.is_enabled())
    ff.quit()
